scripts/wdio/wdio-runner.js

```diff
@@ -19,6 +19,10 @@ async function wdioRunner(options) {
     process.env.BROWSERS = browsers;
   }
 
+  if (theme) {
```

Comment:
I would expect to pass in an array of themes and this would loop it.

Reply:
On second thought, what is the purpose of allowing the test runner to accept the theme environment variable? This env variable affects the theme file we webpack. Unless I am misunderstanding the order, accepting this env in the wdio stage has no effect

Reply:
^^^ I am wrong, take a look at this change https://github.com/cerner/terra-toolkit/pull/324/commits/8068db342003fe483bb3da300c6da640bb091474